Gopkg.toml

```diff
@@ -27,23 +27,23 @@
 
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/apiextensions/pull/287 gets merged.